kafka-message-signing/src/test/kotlin/com/gxf/utilities/kafka/message/signing/TestConstants.kt

```diff
@@ -16,4 +19,14 @@ object TestConstants {
             privateKeyFile = ClassPathResource("/rsa-private.pem"),
             publicKeyFile = ClassPathResource("/rsa-public.pem"),
         )
+
+    fun randomSignature(): ByteBuffer {
```

Comment:
I wouldn't consider this a test constant as it creates a random signature on each call?